### ISSUE TITLE
The use of validate set

### ISSUE THREAD

**Issue by shentt67** [HKUDS/MMSSL#6]
Hi, I notice the datasets are split into the train, validate, and test sets, but the validate set is not used. The model that achieves the best performance on the test set is selected as the best model. I think we should select the best-performed model on validate set, and report the performance on the test set. What's your opinion?

**Comment by weiwei1206** [comment]
The setup for using the validation set test can be done simply by setting is_val of self.test() to ‘True’.
